airflow/providers/slack/notifications/slack.py

```diff
@@ -64,6 +64,8 @@ def __init__(
         proxy: str | None = None,
         timeout: int | None = None,
         retry_handlers: list[RetryHandler] | None = None,
+        unfurl_links: bool = True,
```

Comment:
Is unfurling the default behaviour so far, or is it a breaking change?

Reply:
Good question - both from my experience and the [documentation](https://api.slack.com/reference/messaging/link-unfurling#classic_unfurl) it looks that this is the default behavior

Reply:
Cool

Reply:
https://api.slack.com/methods/chat.postMessage

also double checked in the API reference, seems to be the case ✅